### DISCUSSION TITLE
Complejidad espacial y valores literales

### DISCUSSION THREAD

**Discussion by IoachimusRoderici** [INGCOM-UNRN-P1/cursada-2023#210] [Preguntas y respuestas] [answered]
¿Los valores literales suman a la complejidad espacial de una función, o solamente cuentan las variables?

Por ejemplo en la siguiente función:

```C
int f(int x){
    return x + 10;
}
```

¿Cómo es la complejidad espacial?:
- 1 int por el parámetro `x`
- 2 ints por `x` y por el `10`

**Comment by martinvilu** [comment, marked as answer]
Solo las variables (más argumentos y retorno), el `10` no es un espacio de memoria asignado.
